### PR TITLE
Feature/rfa helpdesk search shared searchbar

### PR DESCRIPTION
**Pull request by rui-palo**

### CODE REVIEW

File: src/stories/searchBarSingleDropdown/searchBarSingleDropdown.stories.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import "../../components/styles/utils.scss";
+
+.rootContainer {
```
Comment (Jonalogy) [inline src/stories/searchBarSingleDropdown/searchBarSingleDropdown.stories.scss]: If this is empty, lets remove it first

File: src/components/searchBarSingleDropdown/searchBarSingleDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            <Input
+              size={Size.Large}
+              type={InputType.Text}
+              value={this.props.inputText}
+              placeholder={this.props.inputPlaceholder}
+              maxLength={this.props.inputMaxlength || 100} 
+              onChange={(e) => {
+                this.props.handleInputChange(e.target.value);
+              }}
+              // TODO pageConst
```
Comment (Jonalogy) [inline src/components/searchBarSingleDropdown/searchBarSingleDropdown.tsx]: Remove this?

File: src/stories/searchBarSingleDropdown/searchBarSingleDropdown.stories.scss (hunk 1 above)
Comment (rui-palo) [reply src/stories/searchBarSingleDropdown/searchBarSingleDropdown.stories.scss]: removed!

File: src/components/searchBarSingleDropdown/searchBarSingleDropdown.tsx (hunk 2 above)
Comment (rui-palo) [reply src/components/searchBarSingleDropdown/searchBarSingleDropdown.tsx]: removed!

**Pull request merged**